website/docs/cloud-docs/architectural-details/data-security.mdx

```diff
@@ -14,12 +14,28 @@ seriously. This table lists which parts of the Terraform Cloud and Terraform Ent
 
 | Object                               | Storage      | Encrypted                |
 | :----------------------------------- | :----------- | :----------------------- |
-| Ingressed VCS Data                   | Blob Storage | Vault Transit Encryption |
-| Terraform Plan Result                | Blob Storage | Vault Transit Encryption |
-| Terraform State                      | Blob Storage | Vault Transit Encryption |
-| Terraform Logs                       | Blob Storage | Vault Transit Encryption |
+| Terraform Configuration (VCS data)   | Blob Storage | Vault Transit Encryption |
+| Private Registry modules             | Blob Storage | Vault Transit Encryption |
```

Comment:
```suggestion
| Private Registry Modules             | Blob Storage | Vault Transit Encryption |
```

I think we actually need to update all of the tables to match our general style guide, but just updating this line to match the style used in the rest of doc.